Description: [PATCH 0005/0006] More specific rules for Debian/Ubuntu
 Add more Ubuntu Changes into the local templates
Author: Stefan Bader <stefan.bader@canonical.com>
Forwarded: no

--- a/examples/apparmor/libvirt-qemu.in
+++ b/examples/apparmor/libvirt-qemu.in
@@ -75,7 +75,6 @@
   # access to firmware's etc
   /usr/share/kvm/** r,
   /usr/share/qemu/** r,
-  /usr/share/qemu-kvm/** r,
   /usr/share/bochs/** r,
   /usr/share/openbios/** r,
   /usr/share/openhackware/** r,
@@ -84,9 +83,6 @@
   /usr/share/seabios/** r,
   /usr/share/misc/sgabios.bin r,
   /usr/share/ovmf/** r,
-  /usr/share/OVMF/** r,
-  /usr/share/AAVMF/** r,
-  /usr/share/qemu-efi/** r,
   /usr/share/slof/** r,
 
   # access PKI infrastructure
@@ -95,7 +91,6 @@
   # the various binaries
   /usr/bin/kvm rmix,
   /usr/bin/qemu rmix,
-  /usr/bin/qemu-kvm rmix,
   /usr/bin/qemu-system-aarch64 rmix,
   /usr/bin/qemu-system-alpha rmix,
   /usr/bin/qemu-system-arm rmix,
@@ -141,7 +136,6 @@
   /usr/bin/qemu-mipsel rmix,
   /usr/bin/qemu-mipsn32 rmix,
   /usr/bin/qemu-mipsn32el rmix,
-  /usr/bin/qemu-nbd rmix,
   /usr/bin/qemu-or32 rmix,
   /usr/bin/qemu-ppc rmix,
   /usr/bin/qemu-ppc64 rmix,
--- a/examples/apparmor/usr.lib.libvirt.virt-aa-helper.in
+++ b/examples/apparmor/usr.lib.libvirt.virt-aa-helper.in
@@ -18,8 +18,6 @@
   owner @{PROC}/[0-9]*/status r,
   @{PROC}/filesystems r,
 
-  /etc/libnl-3/classid r,
-
   # for hostdev
   /sys/devices/ r,
   /sys/devices/** r,
--- a/examples/apparmor/usr.sbin.libvirtd.in
+++ b/examples/apparmor/usr.sbin.libvirtd.in
@@ -13,7 +13,6 @@
   capability sys_admin,
   capability sys_module,
   capability sys_ptrace,
-  capability sys_pacct,
   capability sys_nice,
   capability sys_chroot,
   capability setuid,
@@ -35,7 +34,6 @@
   network inet6 stream,
   network inet6 dgram,
   network packet dgram,
-  network packet raw,
 @@ifge 2009
   network netlink,
 @@end
